incubator/mariadb-cluster/values-production.yaml

```diff
@@ -0,0 +1,216 @@
+## Bitnami MariaDB image
+image: beltran/mariadb:latest
+pullPolicy: Always
```

Comment:
Not sure if it's a good idea on production env. variable. I would use `IfNotPresent`

Reply:
agree, thanks

Reply:
you'd need to use anyways since we're switching to rolling tags for charts :slightly_smiling_face:

Reply:
@sameersbn that's ok for `values.yaml` but we should not use that for `values-production.yaml`

Reply:
With the new `values-production.yaml`, it gets difficult to know which file you're commenting on.

Yes, you are right. For the prod version we should be using a static tag and pull policy `IfNotPresent`. I withdraw my earlier comment